src/Chapter06.Tests/Listing06.31.OverloadingAConstructor.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_31.Tests
+{
+    [TestClass]
+    public class EmployeeTests
+    {
+        [TestMethod]
+        public void EmployeeConstructor_CallConstructorWithTwoParameters_CorrectConstructorCalled()
```

Comment:
This is a long test name where each of the three segments doesn't add much value.